src/store/actions/LoginAction.js

```diff
@@ -1,9 +1,9 @@
-import { GET_USERLOGIN} from "../types";
+import { GET_USERLOGIN, LOGGIN_ERROR} from "../types";
 import axios from 'axios';
 
 export const getLoginUserInstace = (targetuserId) => async dispatch => {
     try {
-        const res = await axios.get(`http://localhost:8000/api/users/users/${targetuserId}/`)
+        const res = await axios.get(`https://631065638105.signin.aws.amazon.com/console/${targetuserId}/`)
```

Comment:
Parameters should accept (uname, pass)

const res = await .... (will need to wait for backend to publish the endpoint, but should be changed to localhost at the moment for testing)